src/OSPSuite.Infrastructure/Workspace.cs

```diff
@@ -46,7 +46,7 @@ public void UpdateJournalPathRelativeTo(string projectFileFullPath)
          if (Journal == null)
             return;
 
-         _project.JournalPath = FileHelper.CreateRelativePath(Journal.FullPath, projectFileFullPath);
+         _project.JournalPath = FileHelper.CreateRelativePath(Journal.FullPath, FileHelper.FolderFromFileFullPath(projectFileFullPath));
```

Comment:
This is really the one change. Before it was behaving as if it was a folder. Now it's correct